src/main.js

```diff
@@ -0,0 +1,52 @@
+class Person {
+  constructor(name, birthday, amount) {
+    this.name = name;
+    this.birthday = birthday;
+    this.initialAmount = amount;
+    this.history = [`Initial: ${this.initialAmount}`];
+  }
+
+  getAge() {
+    const currentDate = new Date();
+    const birthDate = this.birthday.split('.');
+    const months = currentDate.getMonth() - (birthDate[1] - 1);
```

Comment:
it's hard to understand what in `birthDate[n]`

Reply:
Done